frontend/src/i18n/locales/features/landing-page/en-sg.ts

```diff
@@ -0,0 +1,189 @@
+import { LandingPage } from '.'
+
+export const enSG: LandingPage = {
+  hero: {
+    title: 'Build secure government forms in minutes.',
+    subtitle:
+      'Instant, customisable forms with zero code or cost, to safely collect classified and sensitive data.',
+    ctaButtonLabel: 'Start building your form now',
+  },
+  featureSection: {
```

Comment:
I think generally, I wouldn't want to suffix a section with `Section`. Should be obvious enough. At your discretion